### PR TITLE
feature/deseng745: Updated project definition to add project type multi-select values.

### PR DESCRIPTION
**Pull request by jareth-whitney**
- Updated project definition to add project type multi-select values.

### CODE REVIEW

File: api/swagger/swagger.yaml [hunk 1]
```diff
@@ -2629,6 +2651,14 @@ paths:
           description: "The project phase of the file."
           required: false
           type: string
+        - name: projectTypes
```
Comment (Baelx) [inline api/swagger/swagger.yaml]: Why do Documents need a project type? Isn't this field just meant for projects only?

Just from reading the ticket, it looks like you'd need the projectTypes field in 4 places:
- authenticated (/project) get
- auth'd put 
- auth'd post
- public (/public/project) get

File: seed/jsongenerator/projectschema [hunk 2]
```diff
@@ -32,7 +32,8 @@
         proMember               : '{{lorem(1, "words")}}',
         provElecDist            : '{{lorem(1, "words")}}',
         shortName               : '{{lorem(1, "words")}}',
-        projectPhase                  : '{{lorem(1, "words")}}',
+        projectPhase            : '{{lorem(1, "words")}}',
```
Comment (Baelx) [inline seed/jsongenerator/projectschema:35]: Thanks for the due diligence but we don't need to update this!

**Review by Baelx — CHANGES_REQUESTED** [review]
I think the swagger definition for this new field may be incomplete and/or incorrect. Please see my comment in the code!

File: api/swagger/swagger.yaml (hunk 1 above)
Comment (jareth-whitney) [reply api/swagger/swagger.yaml]: Thanks for catching it, just a misplaced field.

**Review by Baelx — APPROVED** [review]
Thanks!

**Pull request merged**